portfolio/src/main/webapp/map.js

```diff
@@ -0,0 +1,47 @@
+// The embedded Google Map for my introduction page
+const map = new google.maps.Map(document.getElementById('map'), {
+  center: {lat: 43, lng: 169},
```

Comment:
Can you make these lat/long values into constant variables eg. CENTER_LATITUDE = 43;

Reply:
Thanks for the reminder!